contents/incentivization.rst

```diff
@@ -0,0 +1,134 @@
+.. _incentivization:
+
+**********************
+Incentives system
+**********************
+
+Cryptoeconomics
+================
+Cryptoeconomics is usually (and somewhat obviously) understood as the intersection of cryptography with economy [https://en.wikiversity.org/wiki/Cryptoeconomics]. The cryptographic part in a broader sense encompasses the vast world of cryptocurrencies, blockchain networks and secure digital decentralized systems, while the economics side generally deals with economic *incentives* for network participant. The basic principle is to reward individual network participants in some way for contributing resources to the network at large, with the aim to  ensure smooth and successful operation of a network, guarantee protocol execution, and fulfill end users' requirements.
```

Comment:
You mean instead of printing the complete link? I don't really think it's always better (I personally like seeing the actual URL, it is clearer where you are being sent to. But I don't mind

Reply:
i like seeing that as well, but i usually opt for taking a look at the lower left corner of the browser.

not a blocker for me anyway :+1: